app/views/content/welcome/landing/_teaching-is-so-rewarding.html.erb

```diff
@@ -55,8 +55,8 @@
       <%= tag.p(subject_specific_quote_data.fetch(:headline), class: "pink") %>
     </div>
 
-    <%= image_pack_tag("static/images/content/hero-images/0017.jpg", alt: "A teacher supporting a pupil with work in the classroom") %>
-    <%= image_pack_tag("static/images/content/hero-images/0004.jpg", alt: "A teacher standing at the front of her classroom teaching a class") %>
+    <%= image_pack_tag("static/images/content/hero-images/teacher2.jpg", alt: "A teacher supporting a pupil with work in the classroom") %>
```

Comment:
These two alt texts shouldn't be set here (they should be in the config file) - but also they relate to the photos being replaced so don't make sense!

Reply:
I've tried removing the alt text and adding into the config file, but it fails the checks when I do. any idea why @martyn-w ?

Reply:
Hi @jenhadfield-dfe I've pushed a small fix to set the alt text for these images from the `images.yml` config file.